PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -470,24 +470,48 @@ DECLARE_SOA_TABLE(DmesonSelection, "AOD", "DINCOLL", // Selection of D meson in
                   aod::hf_selection_dmeson_collision::DmesonSel);
 
 // Note: definition of columns and tables for Electron Hadron correlation pairs
+// definition of columns and tables for electron properties
+namespace hf_electron
+{
+DECLARE_SOA_COLUMN(Phi, phi, float);               //! Phi of electron
+DECLARE_SOA_COLUMN(Eta, eta, float);               //! Eta of electron
+DECLARE_SOA_COLUMN(PtElectron, ptElectron, float); //! Transverse momentum of electron
+DECLARE_SOA_COLUMN(LsElCount, lsElCount, int);     //! like sign Electron
+DECLARE_SOA_COLUMN(UlsElCount, ulsElCount, int);   //! unLike sign Electron
```

Comment:
Replace `...Count` with `n...` and make sure the comments are correct and explicit.